README.md

```diff
@@ -68,7 +68,31 @@ For more information on contributing, please read the [contributing guidelines](
 
 ### Visualisation of the codebase
```

Comment:
This line needs to be removed as it's now on line 93 in the committed version. Sweep remove this accidental duplication